app/scripts/templates/complete_sign_in.mustache

```diff
@@ -0,0 +1,37 @@
+<div id="main-content" class="card">
```

Comment:
This file will have to be renamed `complete_verify_token`.

Ugh, I realize I'm probably the one that started with the `complete_<verb>` filenames, a horrible convention if the routes are named `<verb>_complete`. What was I thinking? I think we should rename all the files, in a separate PR.